alphadia/workflow/optimization.py

```diff
@@ -508,9 +523,9 @@ def _propose_new_parameter(self, df: pd.DataFrame):
 
         """
 
-        return 1.1 * self.workflow.calibration_manager.get_estimator(
+        return self.update_factor * self.workflow.calibration_manager.get_estimator(
```

Comment:
now that we parametrized it, coming back to an earlier suggestion: could we have only one implementation of this?
(this would be a dedicated PR)

Reply:
We could, and I suppose we could always overwrite the default method in a child class if needed. I'll make do this soon.